### PR TITLE
Deleted travis files and updated database.yml

### PR DESCRIPTION
**Pull request by rizzkhan1**
### Context
Cleanup source code and fix inconsistencies with environment files

### Changes proposed in this pull request
Delete Travis build pipeline related files
Delete and rename environment settings files

### CODE REVIEW

File: config/settings/development.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 dfe_signin:
```
Comment (dankmitchell) [inline config/settings/development.yml]: This file is for local development, not the `qa` environment so the localhost URL's need to stay

**Comment by dankmitchell** [comment]
Should this PR also remove the extra environments from https://github.com/DFE-Digital/manage-courses-frontend/blob/master/config/webpacker.yml

File: config/settings/development.yml (hunk 1 above)
Comment (rizzkhan1) [reply config/settings/development.yml]: reverted changes made to development.yml file

**Comment by rizzkhan1** [comment]
> Can we squash on merge 👌

unfortunately squash and merge option isn't enabled on this repo

**Comment by timabell** [comment]
> unfortunately squash and merge option isn't enabled on this repo

it's disabled because you lose a bunch of useful context in the git logs

**Comment by timabell** [comment]
don't forget to delete your branch too

**Pull request merged**